unit_testing/cli/test_cli_sct_deepseg_gm.py

```diff
@@ -1,14 +1,18 @@
-from pytest_console_scripts import script_runner
 import pytest
 import logging
 
+from spinalcordtoolbox.image import Image, compute_dice
+from spinalcordtoolbox.scripts import sct_deepseg_gm
+
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.script_launch_mode('subprocess')
-def test_sct_deepseg_gm_backwards_compat(script_runner):
-    ret = script_runner.run('sct_testing', '--function', 'sct_deepseg_gm')
-    logger.debug(f"{ret.stdout}")
-    logger.debug(f"{ret.stderr}")
-    assert ret.success
-    assert ret.stderr == ''
+@pytest.mark.sct_testing
+@pytest.mark.usefixtures("run_in_sct_testing_data_dir")
+def test_sct_deepseg_gm_check_dice_coefficient_against_groundtruth():
+    """Run the CLI script and verify that dice (computed against ground truth) is within a certain threshold."""
+    fname_out = 'output.nii.gz'
+    fname_gt = 't2s/t2s_uncropped_gmseg_manual.nii.gz'
+    sct_deepseg_gm.main(argv=['-i', 't2s/t2s_uncropped.nii.gz', '-o', fname_out, '-qc', 'testing-qc'])
+    dice_segmentation = compute_dice(Image(fname_out), Image(fname_gt), mode='3d', zboundaries=False)
+    assert dice_segmentation >= 0.85
```

Comment:
i guess the display`# display
    param_test.output += 'Computed dice: '+str(dice_segmentation)
    param_test.output += '\nDice threshold (if computed Dice smaller: fail): '+str(param_test.dice_threshold)` is not important here